fuzz/fsfuzzer.py.data/fsfuzzer.yaml

```diff
@@ -0,0 +1,4 @@
+setup:
+ filesystem : !mux
+  default : 
+   fstype : cramfs|ext2|ext3|ext4|swap|iso9660|msdos|vfat|xfs
```

Comment:
Actually there are more:

```
mksquashfs
hfs
gfs2
jffs2
reiserfs
romfs
ecryptfs
```
These were not supported on my machine, but they are supported by the fsfuzz.

So I'd recommend adding all of them and use `./fsfuzz  --help` to check if the current one is supported during `setUp` and use `self.skip` in case it's not supported.